Automattic/woocommerce-payments
# Add SEPA to storefront checkout as proof of concept

Fixes #1441

### Proof of concept, do not merge

#### Changes proposed in this Pull Request
Proof of concept of how we can add SEPA to Storefront checkout
Almost ready.








#### Testing instructions





* set store currency to euro
* Enable feature flag `wp option update _wcpay_feature_sepa 1`
* Enable sepa in checkbox at bottom of /wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
* Add item to cart and go to checkout
* choose SEPA  direct debit payment option
* enter test IBAN `DE89370400440532013000`
* Place order
* Order should be place and show the PAYMENT METHOD: SEPA Direct Debit
* Order status should be set to on-hold
-------------------
